tests/bundle/EventListener/XmlUnsupportedMediaTypeSubscriberTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Bundle\Rest\EventListener;
+
+use Ibexa\Bundle\Rest\EventListener\XmlUnsupportedMediaTypeSubscriber;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\HeaderBag;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\Exception\UnsupportedMediaTypeHttpException;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+final class XmlUnsupportedMediaTypeSubscriberTest extends TestCase
+{
+    private const XML_REGEXP = '(^application/vnd\.ibexa\.api(\.[A-Za-z]+)+\+xml$)';
```

Comment:
Do we actually have types that contain multiple `.<something>` components?

Outer capture is not necessary at the very least.

```suggestion
    private const XML_REGEXP = '^application/vnd\.ibexa\.api\.[A-Za-z]+\+xml$';
```

Reply:
I actually took `ibexa.rest.output.visitor.json.regexps` pattern and left it untouched. I would personally prefer to have it aligned with all the logic depending on XML/JSON detection.